jelly/pom.xml

```diff
@@ -10,6 +10,15 @@
   <name>Stapler Jelly module</name>
   <description>Jelly binding for Stapler</description>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>commons-lang</groupId>
+        <artifactId>commons-lang</artifactId>
+        <version>2.5</version>
```

Comment:
Maybe better to `<exclude>` it from wherever it was being improperly picked up?

Reply:
I pasted the error in the commit message. So where do you think it is being improperly picked up?

Reply:
Looks like both `htmlunit` and `ezmorph`, with `json-lib` asking for the newest version. Not worth bothering over. (When there is just one dep trail which picks up an old lib, and unfortunately that happens to be the one preferred by Maven’s silly “nearest” algorithm, then it is easier to exclude that trail than to maintain an explicit version number.)

Reply:
> When there is just one dep trail which picks up an old lib, and unfortunately that happens to be the one preferred by Maven’s silly “nearest” algorithm, then it is easier to exclude that trail than to maintain an explicit version number

Yeah, I agree. It gets particularly perverse when Dependabot then starts trying to update that number in the `<dependencyManagement>` section.